akka-persistence/src/main/java/akka/persistence/serialization/MessageFormats.java

```diff
@@ -140,6 +140,16 @@ public static void registerAllExtensions(
      */
```

Comment:
did you re-generate this, with the right protoc version? we have different in master in now

Reply:
I regenerated with 2.5, yes